uwblueprint/ccbc
# [Bug] Get seed data working

## Notion ticket link

[Get seed data working](https://www.notion.so/uwblueprintexecs/16fb020b93f641c8b384f76a1b3c769e?v=aedc8d99cd8b437cb556ece10f3c0e3c&p=a8a384031e1045c389e6c13da4980184)



## Implementation description
* added additional seeding data so website doesn't crash
* added `created_by_id` property to seeding review data !!



## Steps to test
1. Follow steps 1-3, and 6 on here https://github.com/uwblueprint/ccbc#running-migrations--seeding-database
2. Go to website and admin dashboard, for each review, check that book modal and review doesn't crash!



## What should reviewers focus on?
* This assumes that there will always be a user with id 1 in database, since the reviews are created by this user. 

TODO:
- add more seeding data? how much do we want?
- seed staging db => won't this delete all the existing data though?
- where to add wiki on seeding?

## Checklist
